docs/source/intro/install_conda.rst

```diff
@@ -117,13 +129,32 @@ This is the recommended method for users on shared systems and systems where the
        :text: What's next?
        :classes: btn-outline-primary btn-block font-weight-bold
 
+
    ---
 
    **Shut-down services**
 
-   After finishing with your aiida session, particularly if switching between profiles, you may wish to power down the services:
+   After finishing with your aiida session, particularly if switching between profiles, you may wish to power down the daemon and the services:
 
    .. code-block:: console
 
        (aiida) $ verdi daemon stop
-       (aiida) $ pg_ctl stop
+       (aiida) $ pg_ctl -D mylocal_db stop
```

Comment:
I assume this didn't work without the extra argument?

Reply:
Yes, the extra argument is mandatory as it identifies which server needs to be stopped.